django/template/base.py

```diff
@@ -981,6 +981,16 @@ def __repr__(self):
     def render(self, context):
         return self.s
 
+    def render_annotated(self, context):
+        """
+        Special case implementation for performance reasons. The default
+        implementation handles the case where the underlying render() method
+        may throw an exception (of any type) during its execution; however for
+        text nodes specifically, not even a cast is required, so the value
+        may be immediately returned unchecked.
```

Comment:
Maybe:
```suggestion
        """
        Return the given value.

        The default implementation of this method handles exceptions raised
        during rendering, that is not necessary for text nodes.
```

Reply:
@kezabelle What do you think about adjusting a docstring? :point_up:

Reply:
Sorry, yeah I'll do that no problem, will ping once updated. Distracted by going down the rabbit hole elsewhere :)

Reply:
Updated docstring per recommendation, but changed `that` to `which`